### ISSUE TITLE
tune_sr not executed because of file not found when not using "admin" user

### ISSUE THREAD

**Issue by MichelZ** [EMCECS/ECS-CommunityEdition#567]
### Expected Behavior
Works

### Actual Behavior
Does not work :)
File not found for `/home/admin/ECS-CommunityEdition/patches/3.8.0.1/tune_sr`
(The installation continues just fine, but the tuning values are not applied)

### Steps to Reproduce Behavior

1. Install ECS with a user that is NOT named `admin`

### Relevant Output and Logs

---
Notifies: @nikhil-vr

https://github.com/EMCECS/ECS-CommunityEdition/blob/76372ad59ce8f4c5d82d9551b9daea4364188fe6/ui/run.sh#L172
The user is hardcoded here. This should probably be made a relative path?

**Comment by nikhil-vr** [comment]
Will get it fixed , now you could execute this script manually.

**Comment by MichelZ** [comment]
> Will get it fixed , now you could execute this script manually.

Yep, I did that of course once I noticed. I'd like to get it fixed though, as you need to be carefully watching to "catch" the error, so could easily be overlooked. And the settings seem to be important for dealing with space related issues.

Thanks!
